kubernetes/helm_charts/core/nginx-public-ingress/values.j2

```diff
@@ -190,15 +190,15 @@ proxyconfig: |-
       # Enabling caching
       # caching include Accept-Encoding header also, to provide gziped or plain content as per request
       proxy_cache_key "$http_accept_encoding|$request_uri|$request_body";
-      proxy_cache framework_cache;
+      proxy_cache {{proxy_cache_path.medium_cache.keys_zone.split(':') | first}};
       add_header X-Proxy-Cache $upstream_cache_status;
       add_header X-Proxy-Cache-Date $upstream_http_date;
       proxy_cache_use_stale error timeout updating http_500 http_502 http_503 http_504;
       proxy_cache_methods GET HEAD POST;
       proxy_cache_revalidate on;
       proxy_cache_background_update on;
       proxy_cache_lock on;
-      proxy_cache_valid 200 43200;
+      proxy_cache_valid 200 {{proxy_cache_valid.long_validity}};
```

Comment:
If in future, we decide to change this location block from long validity to medium validity, this code needs to be updated. We don't have an option to override this currently.